apps/console/src/extensions/components/password-expiry/components/password-expiry.tsx

```diff
@@ -27,7 +27,8 @@ export const generatePasswordExpiry = (
     componentId: string,
     passwordExpiryEnabled: boolean,
     setPasswordExpiryEnabled: (value: boolean) => void,
-    t: TFunction<"translation", undefined>
+    t: TFunction<"translation", undefined>,
+    isReadOnly: boolean = false
```

Comment:
Let's send the cloud config PR as well.

Reply:
This is not included in cloud extensions.

Reply:
Ack.